README.md

```diff
@@ -148,7 +148,11 @@ module.exports = {
   dist: The sub-directory of the source code where it puts its build artifact. Often "dist".
   */
   localModules: [
-    { moduleName: '@edx/paragon/scss/core', dir: '../src/paragon', dist: 'scss/core' },
+    {
+      moduleName: '@edx/paragon/scss/core',
+      dir: '../src/paragon',
+      dist: 'scss/core',
+    },
```

Comment:
Why only the first element in the list got updated? Also, I think we shouldn't make any changes here, it was fine before when every entry is in one linel